### PR TITLE
Add, Edit, and Remove Activity Group Screen

### LINKED ISSUE #7: Implement Add and Edit Activity Group screen
**Describe the solution you'd like**
Pressing the "Add Activity Group" button on the trip details screen redirects to a new screen to create an activity group.
This screen can also be used to edit an activity group.

Screen offers
- Input fields for name and description
- Header with Return and Save buttons

**Additional context**
None.

IMG_0371

IMG_0355

### LINKED ISSUE #41: Use `KeyboardAvoidingView` for inputs where the keyboard is required
**Describe the solution you'd like**
On iOS, the keyboard sometimes hides the current text field the user types in when the display size is not large enough. We could prevent this by using `KeyboardAvoidingView` (see [React Native docs](https://reactnative.dev/docs/keyboardavoidingview)) where appropriate.

**Additional context**
Please see the screenshots of the `AddTrip` screen, using an iPhone 7 with a 4,7" display. Anything beneath "Test" in the description is hidden by the keyboard.

IMG_2386

IMG_2387

### PR DESCRIPTION
**Pull request by valentingc**
This PR adds a screen to add, edit, and remove activity groups (Closes #7 )
Additionally, a new library is used to fix the keyboard problem we have with Formik. (Closes #41)

Reference Mockup:
image

(No mockup available for removing).

Buttons are at the bottom as the central screen header (#19 ) is not ready yet.
Furthermore, documents were left out currently as discussed.

### CODE REVIEW

File: src/services/i18n.ts [hunk 1]
```diff
@@ -176,6 +235,22 @@ export const resources = {
                 itinerary: {
                     add: " Gruppe Hinzufügen",
                 },
+                addEditActivityGroup: {
+                    titleCreate: "Aktivitätsgruppe hinzufügen",
+                    titleUpdate: "Aktivitätsgruppe aktualisieren",
+                    groupYourActivities: "Gruppiere deine Aktivitäten!😀",
+                    explaination:
```
Comment (whiteclouds7) [inline src/services/i18n.ts]: i think its explanation

File: src/services/i18n.ts [hunk 2]
```diff
@@ -99,6 +121,21 @@ export const resources = {
                 itinerary: {
                     add: "Add Group",
                 },
+                addEditActivityGroup: {
+                    titleCreate: "Add Activity Group",
+                    titleUpdate: "Update Activity Group",
+                    groupYourActivities: "Group your activities!😀",
+                    explaination:
```
Comment (whiteclouds7) [inline src/services/i18n.ts]: I think its explanation

**Review by whiteclouds7 — CHANGES_REQUESTED** [review]
I've tested it and it does what it's supposed to do. Great work. I only found a typo in the i18n, you will also have to change it where you'll use the key outside the file

**Comment by valentingc** [comment]
> I've tested it and it does what it's supposed to do. Great work. I only found a typo in the i18n, you will also have to change it where you'll use the key outside the file

@whiteclouds7 thanks - fixed! 😃

**Review by whiteclouds7 — APPROVED** [review]

**Pull request merged**